go/test/endtoend/vtgate/queries/benchmark/benchmark_test.go

```diff
@@ -127,7 +135,7 @@ func BenchmarkShardedTblNoLookup(b *testing.B) {
 	}
 	for _, rows := range []int{1, 10, 100, 500, 1000, 5000, 10000} {
 		insStmt := tq.getInsertQuery(rows)
-		b.Run(fmt.Sprintf("16-shards-%d-rows", rows), func(b *testing.B) {
+		b.Run(fmt.Sprintf("4-shards-%d-rows", rows), func(b *testing.B) {
```

Comment:
are we creating cluster with 4 shards or 16? I could not see the change to use 16 shards in `main_test`

Reply:
yeah the 16 part did not reflect the test, 4 does.